### PR TITLE
Added Chat with the team part

### PR DESCRIPTION
**Pull request by Rajat-dhyani**
Link to matter most channel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,3 +172,8 @@ Concurrency
 -----------
 
 Thimble uses the [throng](https://www.npmjs.com/package/throng) module to leverage Node's [Cluster API](https://nodejs.org/api/cluster.html) for concurrency. To specify the number of server processes to start set `WEB_CONCURRENCY` to a positive integer value.
+
+Chat with the team
+-----------
+
+You can talk with the team in the [contributing channel]( http://chat.mozillafoundation.org) on [mattermost](https://about.mattermost.com). We're a friendly group!
```
Comment (gideonthomas) [inline README.md]: uppercase `M` in "mattermost"

File: README.md [hunk 2]
```diff
@@ -172,3 +172,8 @@ Concurrency
 -----------
 
 Thimble uses the [throng](https://www.npmjs.com/package/throng) module to leverage Node's [Cluster API](https://nodejs.org/api/cluster.html) for concurrency. To specify the number of server processes to start set `WEB_CONCURRENCY` to a positive integer value.
+
+Chat with the team
```
Comment (gideonthomas) [inline README.md]: Let's maybe change this to `Contact Us` for brevity

File: README.md (hunk 1 above)
Comment (gideonthomas) [inline README.md]: I think we should also change the text to something like:

We're a friendly group, so feel free to chat with us in the "Thimble" channel on Mozilla Chat running on [Mattermost](https://about.mattermost.com). To access Mozilla Chat head to [this link](https://chat.mozillafoundation.org). Note that you will need to create an account first.

You can also download a mobile or desktop client for Mattermost [here](https://about.mattermost.com/download/#mattermostApps).

**Comment by Rajat-dhyani** [comment]
@ryanwarsaw  please check and review the updated version

**Comment by ryanwarsaw** [comment]
The only thing I noticed with this is that the link to Mozilla Chat (chat.mozillafoundation.org) is an http link and not an http**s** one. We automatically redirect people to https so I don't think its an issue.

@gideonthomas Good to merge?

**Comment by Pomax** [comment]
good to go.

**Pull request merged**